docs/api-reference/global-functions/Time.md

```diff
@@ -1,22 +1,23 @@
 ---
-description: Time Time(String time)
+uid: crmscript_global_time
+title: Time
+description: CRMScript Time constructor
 intellisense: Void.Time
 langref: 1
-keywords: Time(String)
+keywords: Time(Time), Time(String)
 so.topic: reference
 ---
 
```

Comment:
Doesn't conform to the standard layout (See [String](https://github.com/SuperOfficeDocs/crmscript/pull/11/files#diff-c72874fdabbc14e3c4e25b587a6d7f229a97f1acce17676d630f97e23cbe6646)).
Considering linking to, or include examples from [data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/time-type.md).